main.css

```diff
@@ -0,0 +1,90 @@
+.container {
+  display: flex;
+}
+
+.container > nav {
+  color: #6070ff;
+  margin: 10px;
+  text-align: center;
+  line-height: 75px;
+  font-size: 30px;
+  font-family: "Poppins";
```

Comment:
- [ ] one of the requirements says that you should stick to the template design by using the same `font family`, `padding` `font size`. Kindly follow the template and use the appropriated font family, the one that you are using here 👆  is not valid. [Here](https://fonts.google.com/specimen/Poppins) the real one.